app/src/main/java/ch/epfl/polybazaar/Utilities.java

```diff
@@ -1,13 +1,38 @@
 package ch.epfl.polybazaar;
 
+import java.util.HashMap;
 import java.util.Map;
 
+import ch.epfl.polybazaar.listing.Listing;
+import ch.epfl.polybazaar.litelisting.LiteListing;
 import ch.epfl.polybazaar.user.User;
 
 public abstract class Utilities {
 
+    public static Map<String, Object> getMap(Object o) {
+        Map<String, Object> result = new HashMap<String, Object>();
+        if (o instanceof Listing) {
+            result.put("description", ((Listing)o).getDescription());
+            result.put("price", ((Listing)o).getPrice());
+            result.put("title", ((Listing)o).getTitle());
+            result.put("userEmail", ((Listing)o).getUserEmail());
+        } else if (o instanceof LiteListing) {
+            result.put("listingID", ((LiteListing)o).getListingID());
+            result.put("price", ((LiteListing)o).getPrice());
+            result.put("title", ((LiteListing)o).getTitle());
+        } else if (o instanceof User) {
+            result.put("email", ((User)o).getEmail());
+            result.put("nickName", ((User)o).getNickName());
+        } else {
+            return null;
+        }
+        return result;
+    }
+
+    // TODO: can be adjusted
     public static boolean nameIsValid(String name) {
-        return (name.matches("[a-zA-Z]+"));
+        //return (name.matches("[a-zA-Z]+"));
+        return true;
     }

```

Comment:
Is this function useful if it always return true?

Reply:
we might want to change it later if we want, that's why i left it.